src/SampleData.jl

```diff
@@ -0,0 +1,112 @@
+
+module SampleData
+
+using CelesteTypes
+import SDSS
+import OptimizeElbo
+import ModelInit
+
+using Base.Test
```

Comment:
Good catch.  I'm fixing this (and responding to your other suggestions) in the unconstrain branch.